gallery/src/components/NxToast/NxToastPage.tsx

```diff
@@ -100,14 +100,26 @@ const NxToastPage = () =>
               <NxTable.Cell>React Element</NxTable.Cell>
               <NxTable.Cell>Yes</NxTable.Cell>
               <NxTable.Cell>
-                A single <NxCode>ReactElement</NxCode> which accepts an <NxCode>onClose</NxCode> prop and which
-                ultimately renders an <NxCode>NxAlert</NxCode> (or one of its variants such as
-                {' '}<NxCode>NxSuccessAlert</NxCode>). Most commonly, this would just be one of the RSC
-                {' '}<NxCode>Nx*Alert</NxCode> components themselves, but custom wrapping components are also permitted.
-                Note that the calling code should not specify the <NxCode>onClose</NxCode> prop for the alert; it will
-                be set up internally by <NxCode>NxToast</NxCode>. Calling code should instead use the
-                {' '}<NxCode>onClose</NxCode> prop of <NxCode>NxToast</NxCode> itself. Additionally, note that the alert
-                text content should be brief: only one rendered line of text per toast is supported.
+                <NxP>
+                  A single <NxCode>ReactElement</NxCode> which accepts an <NxCode>onClose</NxCode> prop and which
+                  ultimately renders an <NxCode>NxAlert</NxCode> (or one of its variants such as
+                  {' '}<NxCode>NxSuccessAlert</NxCode>). Most commonly, this would just be one of the RSC
+                  {' '}<NxCode>Nx*Alert</NxCode> components themselves, but custom wrapping components are also
+                  permitted. Note that the calling code should not specify the <NxCode>onClose</NxCode> prop for the
+                  alert; it will be set up internally by <NxCode>NxToast</NxCode>. Calling code should instead use the
+                  {' '}<NxCode>onClose</NxCode> prop of <NxCode>NxToast</NxCode> itself. Additionally, note that the
+                  alert text content should be brief: only one rendered line of text per toast is supported.
+                </NxP>
+                <NxP>
+                  <NxCode>Nx*Alert</NxCode>s as children of <NxCode>NxToast</NxCode> will be assigned a default role
+                  of <NxCode>alert</NxCode>. The role can be updated to either <NxCode>status</NxCode> or
+                  {' '}<NxCode>log</NxCode>. One of these three {' '}
```

Comment:
We shouldn't bring up `log`.  Even though it makes the test pass I can't see it ever making sense as the role for a toast.  At the same time, I'm seeing no documentation in the standards on the rules that we seem to be encountering here, and even the test tool seems to be a bit inconsistent about it.  As you've observed, `log` works while `marquee` does not.  Presumably, the difference is due to the default `aria-live` values that come with those roles.  However if that's the rule - that it must have an effective `aria-live` value other than `off`, then explcitly setting `aria-live` should also allow it to pass, regardless of the `role`.  In my testing however, it does not.  So at this point we well into the weeds of the undocumented and seemingly inconsistent logic of one particular testing tool - which is not something that we should base our documentation on.

I do still think that `alert` makes sense as the default role for all toast alerts. I think we should document that default, and that it can be overridden, and leave it at that.

Reply:
> I do still think that `alert` makes sense as the default role for all toast alerts. I think we should document that default, and that it can be overridden, and leave it at that.

Sounds good. Updated here: [8cbcf2d](https://github.com/sonatype/sonatype-react-shared-components/pull/766/commits/8cbcf2d165dc3c6dc8c33b35bcb15ae2170f0af9)